benchopt/runner.py

```diff
@@ -401,10 +401,10 @@ def run_benchmark(benchmark, solver_names=None, forced_solvers=None,
 
     # Load the objective class for this benchmark and the datasets
     objective_class = benchmark.get_benchmark_objective()
-    datasets = benchmark.list_benchmark_datasets()
+    datasets = benchmark.datasets
```

Comment:
I would prefer a get_datasets() method as here you have a property that does some compute

Reply:
Let's discuss this tomorrow but I think I prefer the property.